languages/csharp/exercises/concept/classes/.docs/after.md

```diff
@@ -0,0 +1,97 @@
+The primary object-oriented construct in C# is the _class_, which is a grouping of related data ([_fields_][fields]) and behavior ([_methods_][methods]). The fields and methods of a class are known as its _members_.
+
+Access to members can be restricted through access modifiers, the two most common ones being:
+
+- [`public`][public]: the member can be accessed by any code (no restrictions).
+- [`private`][private]: the member can only be accessed by code in the same class.
+
+It is customary to specify an access modifier for all members. If no access modifier is specified, it will default to `private`.
+
+The above-mentioned grouping of related data and behavior plus restricting access to members is known as _encapsulation_, which is one of the core object-oriented concepts.
+
+You can think of a class as a template for creating instances of that class. To [create an instance of a class][creating-objects] (also known as an _object_), the [`new` keyword][new] is used:
+
+```csharp
+class Car
+{
+}
+
+// Create two car instances
+var myCar = new Car();
+var yourCar = new Car();
+```
+
+[Fields][fields] have a type and a name (defined in camelCase) and can be defined anywhere in a class.
+
+```csharp
+class Car
+{
+    // Accessible by anyone
+    public int weight;
+
+    // Only accessible by code in this class
+    private string color;
+}
+```
+
+One can optionally assign an initial value to a field. If a field does _not_ specify an initial value, it wll be set to its type's default value. An instance's field values can be accessed and updated using dot-notation.
```

Comment:
Default values could be linked here: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/default-values

Reply:
Good suggestion!